integration-tests/src/test/java/tachyon/client/TachyonFSIntegrationTest.java

```diff
@@ -95,7 +95,7 @@ public void createFileWithFileAlreadyExistExceptionTest() throws IOException {
     fileId = sTfs.createFile(uri);
   }
 
-  @Test(expected = IOException.class)
+  @Test(expected = IllegalArgumentException.class)
```

Comment:
Actually this is still JUNIT3 way. Can you make it in JUNIT4 way, by using 
```java
@Rule
public ExpectedException mThrown= ExpectedException.none();
```
and in test case, do:
```java
mThrown.expect(IllegalArgumentException.class);
mThrown.expectMessage("what ever the message is.");
```